src/Magnum/Platform/GlfwApplication.h

```diff
@@ -577,6 +605,8 @@ class GlfwApplication {
 
         void setupCallbacks();
 
+        Containers::Array<GLFWcursor*> _cursors{Containers::ValueInit, static_cast<std::size_t>(Cursor::Hand) + 1};
```

Comment:
The size is known at compile time, so this could be a plain C array -- `GLFWcursor* _cursors[6];` (and then you don't need the Array include).

And instead of taking the size from the enum (which would break once someone adds a value after the `Hand`), I'd put `CORRADE_INTERNAL_ASSERT(Int(cursor) < Containers::arraySize(_cursors))` in the setCursor / createCursor methods. That'll also guard it against things like `setCursor(Cursor(0xbad1dea))` :)